CHANGELOG.md

```diff
@@ -29,360 +29,46 @@ Types of changes (Stanzas):
 "Bug Fixes" for any bug fixes.
 "Client Breaking" for breaking Protobuf, gRPC and REST routes used by end-users.
 "CLI Breaking" for breaking CLI commands.
+"Event Breaking" for breaking events.
 "API Breaking" for breaking exported APIs used by developers building on SDK.
 "State Machine Breaking" for any changes that result in a different AppState given same genesisState and txList.
 Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
 
-## [Unreleased]
-
-## [v0.50.1](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.1) - 2023-11-07
-
-> v0.50.0 has been retracted due to a mistake in tagging the release. Please use v0.50.1 instead.
+## [Unreleased](https://github.com/Finschia/finschia-sdk/compare/v0.48.0...HEAD)
```

Comment:
Not from v0.48.1 which is latest version of finschia-sdk?

Reply:
This feature branch is based on main, so it's not from v0.48.1.